common/rdm/RDMAPI.cpp

```diff
@@ -64,6 +64,7 @@ uint8_t RDMAPI::OutstandingMessagesCount(const UID &uid) {
 
 /*
  * @brief Fetch a count of the proxied devices
+ * @param universe the universe which to preform the call on
```

Comment:
This is triggering codespell as it should be perform:
https://travis-ci.org/OpenLightingProject/ola/jobs/433161975

Grammar wise, it should probably say something like "universe the universe to perform the call on"

Reply:
Done